src/file-access.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const errorHandlers = require('./handlers/error-handlers');
+
+const fileRead = (filePath, request, response, onReadSucessCallBack) => {
+    if(fs.existsSync(filePath)){
```

Comment:
don't use sync apis, the point of nodejs is using async non blocking model

Reply:
Is there another way other than using existSync() to check if the file exists, so we can differentiate between not found 404 error and other internal server errors 500 ?